source/GameData.cpp

```diff
@@ -106,7 +106,8 @@ namespace {
 	map<const Sprite *, double> solarWind;
 	Set<News> news;
 	map<string, vector<string>> ratings;
-	
```

Comment:
Add a space between shipCategories and StarField to get this deletion to go away.